mousestyles/path_diversity/path_features.py

```diff
@@ -115,29 +126,35 @@ def compute_angles(path_obj, radian=None):
 
     Examples
     --------
-    >>> path = pd.DataFrame({'t':[2,4.5,10.5], 'x':[0,1,1],\ 
+    >>> path = pd.DataFrame({'t':[2,4.5,10.5], 'x':[0,1,1],\
     'y':[0,0,1], 'isHB':[True,True,False]})
     >>> compute_angles(path)
     [None, 90.0, None]
     """
-    if radian == None:
+    if radian is None:
```

Comment:
Since radian is expected to be a boolean, you could set default radian to be `False` in the function argument and omit these two lines. @togawa28